core/schema.py

```diff
@@ -317,7 +317,10 @@ class Query(graphene.ObjectType):
         MutationLogGQLType, orderBy=graphene.List(of_type=graphene.String))
 
     role = OrderedDjangoFilterConnectionField(
-        RoleGQLType, orderBy=graphene.List(of_type=graphene.String), show_history=graphene.Boolean(),
+        RoleGQLType,
+        orderBy=graphene.List(of_type=graphene.String),
+        is_system=graphene.Boolean(),
```

Comment:
@delcroip here I defined custom filter for that field. And that is why I removed this in GQLType so as to not have the conflict.

Reply:
I need the custom field because default filter from database model filters via "integer" value. In Model of Role we have "is_system" as an integer